tubular/comparison.py

```diff
@@ -0,0 +1,74 @@
+from tubular.base import BaseTransformer
+import pandas as pd
+
+
+class EqualityChecker(BaseTransformer):
+
+    """Transformer to check if two columns are equal.
+
+    This should be used to check if two columns are equal.
```

Comment:
Just saying the same thing twice here

Reply:
changed in: [82645c5](`https://github.com/lvgig/tubular/pull/55/commits/82645c54f664f406abb18540a6cb9ae8158ec3f9`)